tests_and_analysis/test/data/spectrum2d/quartz_fuzzy_map_0.json

```diff
@@ -0,0 +1,267 @@
+{
+    "__euphonic_class__": "Spectrum2D",
+    "__euphonic_version__": "1.3.2+33.gd8680c2.dirty",
+    "metadata": {
+        "common": "yes",
+        "direction": 0
+    },
+    "x_data": [
+        0.0,
+        0.1111111111111111,
+        0.2222222222222222,
+        0.3333333333333333,
+        0.4444444444444444,
+        0.5555555555555556,
+        0.6666666666666666,
+        0.7777777777777777,
+        0.8888888888888888,
+        1.0
+    ],
+    "x_data_unit": "1 / angstrom",
+    "x_tick_labels": [
+        [
+            0,
+            "$\\Gamma$"
```

Comment:
Does this load properly?

Reply:
Looks good 😄 

![Figure_1](https://github.com/user-attachments/assets/fc6049df-ed46-42f0-9228-630fb6e9233a)